### DISCUSSION TITLE
Localization custom noise

### DISCUSSION THREAD

**Discussion by ahmadi-ali** [ctu-mrs/mrs_uav_system#111] [General]
Hi All,
First of all, I have to say thank you for your great work in the simulation of multi UAVs.
I wonder if there would be the possibility of adding custom random noise to the localization of drones to be more realistic to the GPS drift.
Thanks in advance

Regards

**Comment by klaxalk** [comment]
Hey, the simulation already contains random walk as a parameter for the GPS simulation plugin. Feel free to change that to whatever suits you. It is defined individually for each UAV in the mrs_simulation package, e.g., here:
https://github.com/ctu-mrs/mrs_simulation/blob/09a230a62386b8e26ae5245a7663d299c0dd0690/models/mrs_robots_description/urdf/t650.xacro#L476
